programs/solana/programs/ics26-router/src/state.rs

```diff
@@ -80,16 +81,6 @@ pub struct Packet {
     pub payloads: Vec<Payload>,
 }
 
-/// Payload structure
-#[derive(AnchorSerialize, AnchorDeserialize, Clone)]
-pub struct Payload {
-    pub source_port: String,
-    pub dest_port: String,
-    pub version: String,
-    pub encoding: String,
-    pub value: Vec<u8>,
-}
-
```

Comment:
payload should probably still be here, or in another package called `solana_ics26_interface` package, along with most other types defined here